scripts/check_compliance.py

```diff
@@ -668,6 +671,52 @@ def run(self):
             self.add_failure(msg.decode("utf-8"))
 
 
+class PyLint(ComplianceTest):
+    """
+    Runs pylint on all .py files, with a limited set of checks enabled. The
+    configuration is in the pylintrc file.
+    """
+    _name = "pylint"
+    _doc = "https://www.pylint.org/"
+
+    def run(self):
+        self.prepare(ZEPHYR_BASE)
+
+        # Path to pylint configuration file
+        pylintrc = os.path.join(os.path.dirname(__file__), "pylintrc")
```

Comment:
How about ZEPHYR_BASE instead of __file__?

Reply:
`ZEPHYR_BASE` points to the Zephyr root rather than the `scripts` directory in the `ci-tools` repository, so don't think it will work.

Think you're allowed to have the `ci-tools` repository checked out anywhere relative to `ZEPHYR_BASE`.

Reply:
Doh! Yes of course. Sorry for the noise.